lib/r2.rb

```diff
@@ -110,7 +110,7 @@ def minimize(css)
       return '' unless css
 
       css.gsub(/\/\*[\s\S]+?\*\//, '').   # comments
-         gsub(/[\n\r]/, '').              # line breaks and carriage returns
+         gsub(/[\n\r]/, ' ').             # line breaks and carriage returns
```

Comment:
The main goal with removing newlines was to combine R2 with minimizing CSS. By replacing one byte with one byte it removes that space saving. I'm ok with that in favor of correctness but maybe `/[\n\r]/` here could become `/[\n\r]+/` so multiple blank lines become a single space.